iphone/Maps/Classes/MapViewController.mm

```diff
@@ -720,10 +720,7 @@ - (void)switchPositionMode {
 
 - (void)goBack {
    NSString *backURL = [DeepLinkHandler.shared getBackUrl];
```

Comment:
Now this goBack method is called by "Escape" key, and it doesn't check if backURL is empty or valid. Will the app crash if the backurl is missing or is invalid?

I would make a change that returns either nil or valid NSURL from getBackUrl function.

Reply:
I did a test with no backURL and nothing happened, no crash. 

I will make the suggested changes, thank you for your advice.

Reply:
> 

I've made the suggested change, please let me know how it looks. Thank you.